tests/unit/python/foglamp/services/core/api/test_common_ping.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test rest server api for python/foglamp/services/core/api/common.py """
+
+from aiohttp import web
+import json
+import pytest
+
+from foglamp.services.core import routes
+
+
+@pytest.allure.feature("unit")
+@pytest.allure.story("api", "common")
+async def test_ping(test_server, test_client, loop):
+    app = web.Application()
+    # fill route table
+    routes.setup(app)
+
+    server = await test_server(app)
+    server.start_server(loop=loop)
+
+    client = await test_client(app)
+
+    resp = await client.get('/foglamp/ping')
+    assert resp.status == 200
```

Comment:
actual == expected?

Reply:
What do you mean here @ashish-jabble ?

Reply:
Okay, I will change to `assert 200 == resp.status` to keep stuff consistent and as a practice.

Reply:
Here I meant for general test file practice (assert actual == expected)
`E.g. assert 200 = resp.status`